Frontend/Gmfctn/src/app/modules/dashboard/exoft-achievements/exoft-achievements.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import '../../../models/achievement';
+import '../../../models/user';
+import { SaythankModalComponent } from '../../modal-windows/saythank-modal/saythank-modal.component';
+@Component({
+  selector: 'app-exoft-achievements',
+  templateUrl: './exoft-achievements.component.html',
+  styleUrls: ['./exoft-achievements.component.scss', '../../../app.component.scss']
+})
+export class ExoftAchievementsComponent implements OnInit {
+  users: User[] = [{
+                    name: 'Pepe',
+                    surname: 'Jodgi',
+                    total: 10,
+                    icon: '../../../../assets/phoenix.png'
+                  }, {
+                    name: 'Petro',
+                    surname: 'Poroshenko',
+                    total: 100,
+                    icon: '../../../../assets/5.jpg'
+                  }, {
+                    name: 'Shrek',
+                    surname: 'Bolothnyi',
+                    total: 120,
+                    icon: ''
+                  }, {
+                    name: 'Mr.',
+                    surname: 'Heisenberg',
+                    total: 15,
+                    icon: ''
+                  }, {
+                    name: 'Ihor',
+                    surname: 'Da',
+                    total: 190,
+                    icon: ''
+                  }];
+  achList: Achievement[] = [{
+                            icon: '../../../../assets/phoenix.png',
+                            name: 'Exoft turbo power',
+                            description: '',
+                            xp: 5,
+                            time: new Date('December 17, 2005 03:24:00')
+                          }, {
+                            icon: '../../../../assets/phoenix.png',
+                            name: 'Exoft skylark power',
+                            description: '',
+                            xp: 5,
+                            time: new Date()
+                          }, {
+                            icon: '../../../../assets/phoenix.png',
+                            name: 'Exoft corporate power',
+                            description: '',
+                            xp: 5,
+                            time: new Date('March 27, 2021 22:24:00')
+                          }];
+  dictionary: { user: User, achievement: Achievement }[] = [];
```

Comment:
in this block (component) you will display events. each event consists of 
1. user first name
2. user last name
3. user avatar
3. event text
4. event type (can be as enum)
5. added time

you will not get two separate lists of users and achievements. you will get an array of objects with the sctrucutre, described above.

also `dictionary` is not a good name for a property (it is not informative, '`events`' whould be better). also `achList` is not good too - `achievements` is better